editor/src/clj/editor/workspace.clj

```diff
@@ -463,21 +463,24 @@ ordinary paths."
                                                                         path))))]
       (resolve-workspace-resource workspace path))))
 
-(defn- template-path [resource-type]
-  (or (:template resource-type)
-      (some->> resource-type :ext (str "templates/template."))))
+(def ^:private default-user-resource-path "/templates/default.")
+(def ^:private java-resource-path "templates/template.")
 
 (defn- get-template-resource [workspace resource-type]
-  (let [path (template-path resource-type)
-        java-resource (when path (io/resource path))
-        editor-resource (when path (find-resource workspace path))]
-    (or java-resource editor-resource)))
-  
+  (let [resource-path (:template resource-type)]
+    (or
+      ;; default user resource
+      (find-resource workspace (some->> resource-type :ext (str default-user-resource-path)))
+      ;; editor resource provided from extensions
+      (when resource-path (find-resource workspace resource-path))
+      ;; java resource
+      (io/resource (some->> resource-type :ext (str java-resource-path))))))
+
```

Comment:
The priority order changed here.

Previously the template from the editor jar would take precedence over any templates matching the built-in editor file types from extension, but now a `template.sprite` from an extension would win over the one in the editor jar.

Was this intentional?

I think it might be better this way, actually, because it allows an extension to tailor the defaults for newly-created resources to its requirements. For example, if an extension's render script has specific expectations of materials and so on. 🤔